worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightPlatformAdapter.java

```diff
@@ -590,10 +590,11 @@ static void removeBeacon(BlockEntity beacon, LevelChunk levelChunk) {
     }
 
     static List<Entity> getEntities(LevelChunk chunk) {
-        if (PaperLib.isPaper()) {
-            return Arrays.asList(chunk.entities.getRawData());
-        }
-        return chunk.level.entityManager.getEntities(chunk.getPos());
+//        if (PaperLib.isPaper()) {
+//            return Arrays.asList(chunk.getBlockEntities().get(0));
+//        }
+//        return chunk.level.enti.getEntities(chunk.getPos());
```

Comment:
This doesn't look right to me

Reply:
The chunk.entities and level.entityManager fields are removed, I was trying to find alternative field and forgot to undo it before commenting.
I fixed it.